src/pathfinding_service/claim_fees.py

```diff
@@ -0,0 +1,132 @@
+import sys
+from typing import Iterable, Tuple
+
+import click
+import structlog
+from web3 import Web3
+from web3.contract import Contract
+
+from pathfinding_service.database import PFSDatabase
+from pathfinding_service.model import IOU
+from raiden.utils.typing import BlockNumber
+from raiden_contracts.constants import CONTRACT_ONE_TO_N
+from raiden_contracts.contract_manager import ContractManager, contracts_precompiled_path
+from raiden_libs.cli import blockchain_options, common_options, connect_to_blockchain
+from raiden_libs.types import Address
+from raiden_libs.utils import private_key_to_address
+
+log = structlog.get_logger(__name__)
+
+
+@click.command()
+@click.option(
+    '--rdn-per-eth',
+    default=0.0025,
+    type=float,
+    help='Current RDN/ETH price, used to check claimed amount > transaction cost',
+)
+@click.option(
+    '--expires-within',
+    default=4 * 60 * 24 * 7,  # one week
+    type=click.IntRange(min=1),
+    help='Only IOUs which expire withing this number of blocks will be claimed',
+)
+@common_options('raiden-monitoring-service')
```

Comment:
I guess this should be pathfinding service

Reply:
Yes, good catch!